setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import find_packages
+import os
+from setuptools import setup, Command
+
+
+class CleanCommand(Command):
+    """Custom clean command to tidy up the project root."""
+    user_options = []
+    def initialize_options(self):
+        pass
+    def finalize_options(self):
+        pass
+    def run(self):
+        os.system('rm -vrf build  *.egg-info')
+
+
+setup(
+	name='GP-Optimization',
+	version='1.0.0',
+	description='GP optimization project',
+	author='Bosak Adam',
+	author_email='bosadam@seznam.cz',
+	url='https://github.com/bosakad/GP-Optimization',
+	packages=find_packages(),
+	cmdclass={
+        'clean': CleanCommand,
+    }
+
+)
```

Comment:
Please add the requirements from `requirements.txt` to `setup.py`. They should go to

```python
install_requires=[
        "cvxpy>=1.2.1",
        "cycler>=0.11.0",
        "numpy>=1.22", # and so on, please write all the packages you need here
    ],
```

Please note that I write `>=`. It is generally better to have a flexibility in packages rather that require very paricular version. You never know what changes those guys in Numy or Mosek may do so a particular version becomes deprecated or so. So we should keep upper bound open :)